web/src/pages/profile.tsx

```diff
@@ -113,11 +110,6 @@ const ProfilePage = (props: Props): ReactElement => {
 
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   const redirect = (params?: { [key: string]: any }, routerType = router.push): Promise<boolean> => {
-    if (checkQueryValue(router, QUERIES.path, "businessFormation")) {
-      const formationUrlSlug = getTaskFromRoadmap(roadmap, formationTaskId)?.urlSlug ?? "";
```

Comment:
I think we should be using this getTaskFromRoadmap function in the code above that changed in BusinessStep and such. That way, the URL can get changed by the content team, and the code will still work because it's dynamically looking up the url from the roadmap